resources/views/admin/sleeps/create.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('layouts.default')
+
+@section('title')
+Add Sleep
+@parent
+@stop
+
+
+
+@section('content')
+<div class="content-wrapper">
+    <!-- Main content -->
+    <section class="content">
+        <div class="row">
+            <div class="col-lg-12 col-md-12">
```

Comment:
use col-12 instead

Reply:
we are using bootstrap 3 in our boiler template, that's why we are using `col-lg-12 col-md-12`

Simple `col-12` cause issues in bootstrap 3